frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -70,21 +70,29 @@ impl TypeDartGeneratorTrait for TypeEnumRefGenerator<'_> {
             .map(|(idx, variant)| {
                 let args = match &variant.kind {
                     IrVariantKind::Value => "".to_owned(),
-                    IrVariantKind::Struct(st) => st
-                        .fields
-                        .iter()
-                        .enumerate()
-                        .map(|(idx, field)| {
-                            let val =
-                                format!("_wire2api_{}(raw[{}]),", field.ty.safe_ident(), idx + 1);
-                            if st.is_fields_named {
-                                format!("{}: {}", field.name.dart_style(), val)
-                            } else {
-                                val
-                            }
-                        })
-                        .collect::<Vec<_>>()
-                        .join(""),
+                    IrVariantKind::Struct(st) => {
+                        let st = st
+                            .fields
+                            .iter()
+                            .enumerate()
+                            .map(|(idx, field)| {
+                                let val = format!(
+                                    "_wire2api_{}(raw[{}]),",
+                                    field.ty.safe_ident(),
+                                    idx + 1
+                                );
+                                if st.is_fields_named {
+                                    format!("{}: {}", field.name.dart_style(), val)
+                                } else {
+                                    val
+                                }
+                            })
+                            .collect::<Vec<_>>();
+                        if self.ir.is_exception {
+                            //st.push("raw['backtrace']".to_string());
+                        }
+                        st.join("")
```

Comment:
nit: Guess that we are no longer doing anything special here, so the old code just works and we can revert to it?